react/features/virtual-background/components/VirtualBackgroundDialog.js

```diff
@@ -79,38 +158,84 @@ function VirtualBackground({ dispatch, t }: Props) {
             submitDisabled = { false }
             titleKey = { 'virtualBackground.title' }
             width = 'small'>
-            <div className = 'virtual-background-dialog'>
-                <Tooltip
-                    content = { t('virtualBackground.removeBackground') }
-                    position = { 'top' }>
-                    <div
-                        className = { selected === 'none' ? 'none-selected' : 'virtual-background-none' }
-                        onClick = { () => removeBackground() }>
-                        None
+            {loading ? (
+                <div>
+                    <span>{t('virtualBackground.pleaseWait')}</span>
+                    <Spinner
+                        isCompleting = { false }
+                        size = 'medium' />
+                </div>
+            ) : (
+                <div>
+                    <div className = 'virtual-background-dialog'>
+                        <Tooltip
+                            content = { t('virtualBackground.removeBackground') }
+                            position = { 'top' }>
+                            <div
+                                className = { selected === 'none' ? 'none-selected' : 'virtual-background-none' }
+                                onClick = { removeBackground }>
+                                None
```

Comment:
Apply a translation to this. Or maybe we can use an icon to not translate it at all.